### PR TITLE
Fontscale ctrl+click

### PR DESCRIPTION
**Pull request by jeremy-rifkin**
This PR makes ctrl+click on the font scale widget reset the font scale to default. I also updated the tooltip for the button.

There are a couple of formatting changes here not done by me, I think it was automatically done by prettier?

### CODE REVIEW

File: static/widgets/fontscale.ts [hunk 1]
```diff
@@ -66,11 +64,26 @@ function makeFontSizeDropdown(elem: JQuery, obj: FontScale, buttonDropdown: JQue
         }
         elem.children().eq(selectedId).trigger('click');
     });
+
+    // ctrl+click handler
+    buttonDropdown.on('click', e => {
+        if (e.ctrlKey) {
+            // This is a hack. It prevents bootstrap's click listener from opening the dropdown
```
Comment (RubenRBS) [inline static/widgets/fontscale.ts:71]: Not a hack if it's the only way to avoid the default behaviour right?

**Review by RubenRBS — APPROVED** [review]
Love the idea. Seems like I wont be the only one to use the scrowheel feature anymore heh

File: static/widgets/fontscale.ts (hunk 1 above)
Comment (jeremy-rifkin) [reply static/widgets/fontscale.ts:71]: Haha fair enough :) I wish there was a better way to do this, I don't think it has too much potential to cause problems down the road at least

**Comment by RubenRBS** [comment]
Thanks a lot! :)

**Comment by jeremy-rifkin** [comment]
Thanks for merging!

**Pull request merged**